src/wexond/main/view.ts

```diff
@@ -1,14 +1,198 @@
 import {
   BrowserView,
-  app,
+  // app,
   Menu,
   nativeImage,
   clipboard,
-} from 'electron';
-import { appWindow } from '.';
-import { sendToAllExtensions } from './extensions';
-import { engine, isShieldToggled } from './services/web-request';
-import { parse } from 'tldts';
+  // BrowserWindow,
+  // BrowserViewConstructorOptions,
+} from 'electron'
+import { appWindow } from '.'
+import { sendToAllExtensions } from './extensions'
+import { engine } from './services/web-request'
+import { settings } from './index'
+import { parse } from 'tldts'
+import { buildPreloadPath } from '../../common/preload'
+
+// function BrowserWindowExtend(electronClass: BrowserWindow) {
```

Comment:
Is this big chunk of commented code needed ?

Reply:
No it's not, I did not push the message I wrote earlier but it's going to be removed.

It's related to the fact that extending electron's base classes is not supported, so I wanted to create a simple class that would implement the exact same interface and just act as a bridge. Might be needed in the future but overkill for now.